indico/htdocs/js/indico/modules/registration/registration.js

```diff
@@ -108,6 +108,20 @@
         $('#select-none').on('click', function() {
             $('table.i-table input.select-row').prop('checked', false).trigger('change');
         });
+
+        $('.js-dialog-send-email').ajaxDialog({
+            onClose: function(data) {
+                if (data) {
+                    location.reload();
```

Comment:
right, we don't need a reload here. How can I show a flash message without reloading?